cmd/elastic_ip_delete.go

```diff
@@ -60,25 +60,15 @@ func (c *elasticIPDeleteCmd) cmdRun(_ *cobra.Command, _ []string) error {
 		}
 	}
 
-	err = decorateAsyncOperations(fmt.Sprintf("Deleting Elastic IP %s...", c.ElasticIP), func() error {
+	return decorateAsyncOperations(fmt.Sprintf("Deleting Elastic IP %s...", c.ElasticIP), func() error {
 		op, err := client.DeleteElasticIP(ctx, elasticIP.ID)
 		if err != nil {
 			return fmt.Errorf("exoscale: error while deleting Elastic IP: %w", err)
 		}
 
 		_, err = client.Wait(ctx, op, v3.OperationStateSuccess)
-		if err != nil {
-			return fmt.Errorf("exoscale: error while waiting for Elastic IP deletion: %w", err)
```

Comment:
As I pointed out in another PR, if you decided to add more context on the error you can keep, it, I was pointing the change with no context added.

To stay consistent, everywhere, it's better if you not do a mix of sometime add error wrapping and sometime not, you decide, you wrap all and return the proper error, or you wrap not.

IMO, having error wrapping with more context is always better in case of error :) but make sure to respect the same behavior everywhere and stay consistent